apps/condo/domains/meter/utils/metersImporters/SbbolMetersImporter.ts

```diff
@@ -0,0 +1,125 @@
+import { RegisterMetersReadingsReadingInput } from '@app/condo/schema'
+
+import {
+    COLD_WATER_METER_RESOURCE_ID,
+    ELECTRICITY_METER_RESOURCE_ID,
+    GAS_SUPPLY_METER_RESOURCE_ID,
+    HEAT_SUPPLY_METER_RESOURCE_ID,
+    HOT_WATER_METER_RESOURCE_ID,
+} from '@condo/domains/meter/constants/constants'
+
+import { AbstractMetersImporter } from './AbstractMetersImporter'
+
+const METERS_BLOCK_LENGTH = 7
+
+export default class SbbolMetersImporter extends AbstractMetersImporter {
+
+    protected transformRow (row: string[]): RegisterMetersReadingsReadingInput[] {
+        /**
+         * Each meter fields. Order is important.
+         * @see https://vgkh.ru/faq-1s-zhkh/obmen-sberbank/obmen-s-sistemoj-sberbank-biznes-onlajn/
+         */
+        const [accountNumber, clientName, accountGlobalId, addressGlobalId, address, ...metersBlocks] = row
+        const result = {}
+        let nextBlock = metersBlocks.splice(0, METERS_BLOCK_LENGTH)
+        while (nextBlock.length === METERS_BLOCK_LENGTH) {
+            const [meterType, meterNumber, meterName, nextVerificationDate, meterValue, date, additionalInfo] = nextBlock
+
+            if (meterType && meterNumber && meterValue && date) {
+                let values, meterResourceId, numberOfTariffs = 1
+
+                /**
+                 * @see https://vgkh.ru/faq-1s-zhkh/obmen-sberbank/obmen-s-sistemoj-sberbank-biznes-onlajn/#shablon-pokazaniya-schetchikov
```

Comment:
i'm afraid of this link. maybe place it inside our documentation?

Reply:
Removed